src/store/flow/block-types/Core_LogBlockStore.ts

```diff
@@ -12,13 +12,11 @@ export const getters: GetterTree<IFlowsState, IRootState> = {}
 
 export const mutations: MutationTree<IFlowsState> = {}
 export const actions: ActionTree<IFlowsState, IRootState> = {
-
+  async editMessage({commit}, {blockId, message}: { blockId: string, message: string }): Promise<string> {
+    commit('flow/block_updateConfig', {blockId, newConfig: {message}}, {root: true})
```

Comment:
This seems dangerous to me that we overwrite the entire config instead of modifying the single key the method indicates! Can we change this to only modify the message key instead? I'm just imagining that as a pattern, this would become a bug if we tried to add any other properties to the LogBlock's config interface. What do you think?

Reply:
Good point. Fixed!